lookout/style/format/utils.py

```diff
@@ -0,0 +1,17 @@
+import cProfile
+import pstats
+import io
+
+
+def profile(func):
+    def wrapped_func(*args, **kwargs):
```

Comment:
functools.wraps is missing

Reply:
Also the name should be `wrapped_profile`. The reason is that if you debug somewhere inside `func`, you will otherwise see "wrapped_func" in the call stack which is very unobvious in case there is more than one decorator.